podman_compose.py

```diff
@@ -1284,7 +1333,12 @@ def normalize_service(service, sub_dir=""):
         if is_str(extends):
             extends = {"service": extends}
             service["extends"] = extends
-    return service
+
+    try:
+        tagObj.value = serivce
```

Comment:
Is this a typo?

Reply:
No you need to still normalize the values of the Override-tag to be valid in the end (as they are the ones used) but pass the override-tag through to the recursive merge to do the actual override. I used try to determinate if there is a tagObj defiend and if so get the normalized values to the tagObj and return it otherwise return the normalized service-definition.

Reply:
Also this isn't my most recent version anymore as I'm locally working an a version based on the latest https://github.com/maurerle/podman-compose but for now using docker-compose again.

Reply:
`serivce` seems to be non-existing variable though.

Reply:
It is, I'm not coming out of the python-space so I searched how to return it if set and another if not

Reply:
Edit: You are all right - missed the typo but that's not an issue/the latest state and needs a rebase.

How likely are my chances to get this approach merged, if I resubmit it?

Reply:
Good question, haven't looked into viability of the PR so far. Could you point me to the part of the compose spec that this implements? I didn't find with couple minutes of googling and better I don't make an error here :)

Overall I think podman-compose should implement all things possible in non-swarm compose that can be implemented on podman. We may argue about the approach of implementation though.

What I will definitely ask that's not part of the PR right now is unit and integration tests.